demisto_sdk/commands/download/tests/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
why do you need this file?

Reply:
https://github.com/demisto/demisto-sdk/commit/31691217bb9d611c7a48863a17549bb25a1a582c